test/test.py

```diff
@@ -22,6 +22,8 @@
 from hca import constants  # noqa
 from test.config import override_oauth_config  # noqa
 
+dirpath = os.path.dirname(os.path.realpath(__file__))
```

Comment:
Was going to refactor for all tests using `dirpath` to just use this but didn't want to clutter the pr. Will just move to `test_get_files_cli` for now.